packages/components/src/components/switch/switch.scss

```
@@ -76,7 +76,7 @@ $checked-active-transition-size: calc(
 
 		// thumb
 		&::before {
-			content: "O";
+			content: "°";
```

```suggestion
			content: "";
```

Do we actually need a character in here at all? Most likely it won't become part of the accessible name of this form element, but you never know especially regarding future developments.

---

Yes we need this. It is a workaround for `forced-color` mode from chrome